enes-homework.js

```diff
@@ -0,0 +1,145 @@
+//Q1
+
+const Turkish = 'Merhaba Dunya',
+  English = 'Hello, World',
+  French = 'Bonjour, le Monde',
+  Dutch = 'Hoi, de wereld';
+console.log(Turkish, English, French, Dutch);
+
+//Q2
+console.log('I'm awesome'); 
+//here quoting is problem. Two solutions: using double quote/backsticks or escape element. So;
+console.log('I\'m awesome');
+console.log("I'm awesome");
+
+//Q3
+let x;
+console.log("the value of x will be : undefined");
+console.log(x);
+x = 7;
+console.log("the value of x will be : 7");
+console.log(x);
+
+//Q4
+const y = "I like birds";
+console.log("the value of y will be : I like birds");
+console.log(y);
+y = "I hate birds";
+console.log("the value of y will be : error because I have used const as a variable. The case will be different if I had used var or let. While debugging it does not throw an error message I don't know why.");
+console.log(y);
+
+//Q5 (There are two solutions possible. Either using if/else or Math.round() method)
+//Method1
+let z = 7.25;
+console.log(z);
+let a = z - 0.5;
+if (a<7) {
+  a=7
+}
+else {
+  a=8
+};
+console.log(a);
+
+//Method2
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+
+//The rest is same;
+if (z - a > 0) {
+  console.log(z)
+}
+else {
+  console.log(a)
+}
+
+//Q6
+
+const emptyArrOfAnimals = [];
+console.log("emptyArrOfAnimals : empty array");
+console.log(emptyArrOfAnimals);
+
+const myFavouriteAnimals = ["Budgie Parrots", "German Shephard", "Goat"];
+console.log(myFavouriteAnimals);
+myFavouriteAnimals[3] = "baby pig"; //myFavouriteAnimals.push("baby pig") method can also be used
+console.log(myFavouriteAnimals);
+
+//Q7
+
+let myString = "this is a test";
+console.log(myString);
+console.log(myString.length);
+
+//Q8 
+
+let firstVar = 112;
+let secondVar = "ball";
+let thirdVar = true;
+let fourthVar = "112";
+
+console.log(firstVar, secondVar, thirdVar, fourthVar);
+console.log("typeof firstVar:number; typeof secondVar: string; typeof thirdVar: booelan; typeof fourthVar: string")
+
+typeof firstVar;
+typeof secondVar;
+typeof thirdVar;
+typeof fourthVar;
+
+typeof firstVar === typeof secondVar;
+typeof thirdVar !== typeof fourthVar;
+typeof secondVar === typeof fourthVar;
```

Comment:
If you remember from our class today when we learnt about statements and expressions, lines 90 to 92 are expressions which evaluate into a `true` or a `false` value. However, we don't do anything with these values. It would make more sense if you perhaps `console.log` these values. 🙂